extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/DevConsoleCORSFilter.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.vertx.http.runtime.devmode;
+
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.vertx.http.runtime.cors.CORSConfig;
+import io.quarkus.vertx.http.runtime.cors.CORSFilter;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.ext.web.RoutingContext;
+
+public class DevConsoleCORSFilter extends CORSFilter {
+
+    private static final String LOCAL_HOST = "localhost";
+    private static final String HTTP_LOCAL_HOST = "http://" + LOCAL_HOST;
+    private static final String HTTPS_LOCAL_HOST = "https://" + LOCAL_HOST;
+
+    public DevConsoleCORSFilter() {
+        super(corsConfig());
+    }
+
+    private static CORSConfig corsConfig() {
+        CORSConfig config = new CORSConfig();
+        config.origins = Optional.of(List.of(HTTP_LOCAL_HOST, HTTPS_LOCAL_HOST));
```

Comment:
Doesn't the origin require the port as well?

Reply:
@stuartwdouglas Yeah... So, I'll need to get hold of `quarkus.http.port`, as `8080` won't be guaranteed...

Reply:
@stuartwdouglas I had to update it to avoid extending and now it is delegating to CORS filter given that this property is runtime init, but creating CORS filter is very cheap so should be fine.